### PR TITLE
Update qihoo360 and geolocation-cn

### PR DESCRIPTION
**Pull request by EpLiar**

### CODE REVIEW

File: data/qihoo360 [hunk 1]
```diff
@@ -1,15 +1,22 @@
 360.com
-360.cn
```
Comment (iochen) [inline data/qihoo360:2]: why remove this domain

Comment (EpLiar) [reply data/qihoo360:2]: Because `qihoo360` is included in `geolocation-cn`. Remove the `.cn` domain according to `tld-cn`.

Comment (iochen) [reply data/qihoo360:2]: If so, how can I block qihoo360?

Comment (EpLiar) [reply data/qihoo360:2]: OK, I see what you mean.

**Pull request closed without merging**